src/ui/Omnibox/Omnibox.ts

```diff
@@ -260,7 +262,12 @@ export class Omnibox extends Component {
     querySuggestCharacterThreshold: ComponentOptions.buildNumberOption({
       defaultValue: 0,
       min: 0
-    })
+    }),
+
+    /**
+     * Specify wether all active query filters should be cleared everytime the user inputs a new query.
```

Comment:
Setting this to `true` would not clear the filter expression enforced by the currently selected tab though, correct?

Essentially, it clears the breadcrumbs?

If so, I would say:

> Whether to clear all active query filters when the end user submits a new query from the search box.
>
> **Note:** This does not include the filter expression enforced by the currently selected tab, if any.
>
> **Default:** `false`